src/app/ApplicationTemplate.Shared.Views/Behaviors/FormattingTextBoxBehavior.cs

```diff
@@ -195,7 +195,7 @@ private static void FormatText(TextBox textbox, string textFormat)
 			return;
 		}
 
-		_ = textbox.Dispatcher.RunAsync(CoreDispatcherPriority.Normal, () =>
+		_ = textbox.Dispatcher?.RunAsync(CoreDispatcherPriority.Normal, () =>
```

Comment:
You should be using DispatcherQueue instead of Dispatcher since .Net 6.
Dispatcher is always going to be null so that code will never run like this.

Reply:
fixed